### ISSUE TITLE
Update Bindings API to GodotSteam 4.11

### ISSUE THREAD

**Issue by PhilippFors** [LauraWebdev/GodotSteam_CSharpBindings#41]
Godotsteam recently had a new update so some of the C# Bindings are out of date.
Would be great to be able to use the latest GodotSteam version and CSharpbindings.
Not sure if you add them by hand or if you autogenerate the bindings already.

**Comment by plyoung** [comment]
This might be why `Steam.ExchangeItems` is incorrect? It is supposed to take arrays for all params but in the C# version the last 3 params take uint/ulong.

**Comment by PhilippFors** [comment]
haven't looked at that at all but I'm looking to update the bindings for myself as of right now.

**Comment by plyoung** [comment]
At least it is easy to "patch". I've just added this function to my own steam related class and it works. 

```
private static int ExchangeItems(long[] outputItems, int[] outputQuantity, long[] inputItems, int[] inputQuantity)
{
	return Steam.GetInstance().Call(Methods.ExchangeItems, outputItems, outputQuantity, inputItems, inputQuantity).AsInt32();
}
```

**Comment by PhilippFors** [comment]
yeah since it's all handwritten anyway, implementing all the changes should be trivial
